### DISCUSSION TITLE
Windows Server 2019 and "IMAGE_STATE_UNDEPLOYABLE" issue

### DISCUSSION THREAD

**Discussion by runnerImages** [actions/virtual-environments#7745] [General questions] [answered]
### Description

Hello,

Image build ends up with "image_state_undeployaable" message.

==> vhd: Machine successfully restarted, moving on
==> vhd: Provisioning with Powershell...
==> vhd: Provisioning with powershell script: /tmp/powershell-provisioner867035536
    vhd: IMAGE_STATE_COMPLETE
    vhd: IMAGE_STATE_UNDEPLOYABLE
    vhd: IMAGE_STATE_UNDEPLOYABLE
    vhd: IMAGE_STATE_UNDEPLOYABLE
    vhd: IMAGE_STATE_UNDEPLOYABLE
    vhd: IMAGE_STATE_UNDEPLOYABLE
    vhd: IMAGE_STATE_UNDEPLOYABLE
    vhd: IMAGE_STATE_UNDEPLOYABLE

I tried rebuilding image many times (sometimes getting other errors) but this is the error which is re-occuring on regular basis.

### Platforms affected

- [X] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] macOS 13
- [X] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

Current image version: '11220204'

### Is it regression?

no

### Expected behavior

Sysprep completed without errors

### Actual behavior

Build fails on sysprep

### Repro steps

Run build creation

**Comment by ilia-shipitsin** [comment]
do you run in under some CI ? is it public ?
can you provide more repro details ?

**Reply by runnerImages** [reply]
My CI environment is not public. I'm using fork of the main repo https://github.com/runnerImages/runner-images with checkout from ADO pipeline using an GitHub connection.

Could you please let me know what information will be useful for troubleshooting so I can collect them and provide.

Regards,
Lukasz

**Comment by mikhailkoliada** [comment]
I think it has been discussed here https://github.com/actions/runner-images/discussions/6251

**Reply by runnerImages** [reply]
Thank you for sharing it. I have reviewed this discussion before and applied some mitigation but the issue was not resolved.
In my environment the DfC is configured using Policies. I have applied policy exemption for "Configure Azure Defender for servers to be enabled" and "Configure machines to receive a vulnerability assessment provider". 
Shall I disable DfC to resolve this issue?

**Reply by vpolikarpov-akvelon** [reply, marked as answer]
I would recommend to review sysprep logs. This may be the only way to understand what makes Sysprep execution unsuccessful.
Common locations are listed here: https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep-process-overview#sysprep-log-files
